crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -171,3 +206,76 @@ pub struct Capsule {
 }
 impl super::Primitive2d for Capsule {}
 impl Primitive3d for Capsule {}
+
+impl Capsule {
+    /// Create a new `Capsule` from a radius and half-length
+    pub fn new(radius: f32, half_length: f32) -> Self {
+        Self {
+            radius,
+            half_length,
+        }
+    }
+}
+
+/// A cone primitive.
+#[derive(Clone, Copy, Debug)]
+pub struct Cone {
+    /// radius of the base
+    pub radius: f32,
+    /// height of the cone
+    pub height: f32,
```

Comment:
In this case, cones are more naturally represented by a height.

Reply:
Yep, almost every cone representation seems to use a full height and a radius or angle. The origin is usually at the base or tip, the comments should probably tell which one it is, otherwise we're gonna get funny situations where one developer uses a different origin than another :thinking: